### PR TITLE
Pre-v2 branching code cleanup

### PR DESCRIPTION
**Pull request by drwill-ms**
Making non-breaking changes that apply to both v1 and v2 to cut down on rebasing hell when starting an active v2 feature branch. This is just a subset of all projects.

### CODE REVIEW

File: common/src/device/provisioning/transport/X509CertificateInfo.cs [hunk 1]
```diff
@@ -51,41 +51,49 @@ public X509CertificateInfo()
...
         public string Sha1Thumbprint { get; set; }
 
         /// <summary>
+        /// The certificate SHA256 thumbprint.
         /// </summary>
         [JsonProperty(PropertyName = "sha256Thumbprint")]
         public string Sha256Thumbprint { get; set; }
 
         /// <summary>
+        /// The certfificate issuer name.
```
Comment (timtay-microsoft) [inline common/src/device/provisioning/transport/X509CertificateInfo.cs:72]:
```suggestion
        /// The certificate issuer name.
```

File: common/src/device/provisioning/transport/X509CertificateInfo.cs [hunk 2]
```diff
@@ -51,41 +51,49 @@ public X509CertificateInfo()
...
         /// </summary>
         [JsonProperty(PropertyName = "issuerName")]
         public string IssuerName { get; set; }
 
         /// <summary>
+        /// The certfificate invalidity before date in UTC.
```
Comment (timtay-microsoft) [inline common/src/device/provisioning/transport/X509CertificateInfo.cs:78]:
```suggestion
        /// The certificate invalidity before date in UTC.
```

File: provisioning/transport/amqp/src/SaslTpmHandler.cs [hunk 3]
```diff
@@ -267,7 +264,7 @@ private enum SaslChallengeAction
         {
             First,
             Interim,
-            Final
+            Final,
```
Comment (timtay-microsoft) [inline provisioning/transport/amqp/src/SaslTpmHandler.cs:267]: I don't expect this enum to grow any further, for what it's worth.

**Review by timtay-microsoft — APPROVED** [review]

File: provisioning/transport/amqp/src/SaslTpmHandler.cs (hunk 3 above)
Comment (drwill-ms) [reply provisioning/transport/amqp/src/SaslTpmHandler.cs:267]: Yeah, I just style them this way as a rule, for consistency.

**Pull request merged**